lib/extended-css.js

```diff
@@ -143,12 +145,23 @@ var ExtendedCss = function (styleSheet) { // jshint ignore:line
         affectedElement.node.style.cssText = affectedElement.originalStyle;
     };
 
+    /**
+     * Warn if `applyRule` call takes more than this in milliseconds.
+     */
+    var WARNING_THRESHOLD = 50;
```

Comment:
1. If `debug` is enabled that way, it'd be better to simply print the information every time.
2. Also, I'd like to have an option to enable debug globally for all selectors. This way I can keep this global debugging rule in the user filter and keep an eye on the console.